docs/custom_directives.py

```diff
@@ -3,12 +3,6 @@
 from docutils import nodes
 import re
 import os
-import sphinx_gallery
-
-try:
-    FileNotFoundError
-except NameError:
-    FileNotFoundError = IOError
```

Comment:
Leftover Python 2 code, I believe.

Reply:
Although this is no different than the other repos in that regard. Guess you noticed it here.

Reply:
> Guess you noticed it here.

Haha precisely. My process has been to go one project at a time cleaning up these CSS files, tracked by this checklist at https://github.com/Qiskit/qiskit_sphinx_theme/issues/262#issuecomment-1544476510. The further along I get in these update-every-single-project-style changes, I usually notice a slightly better way to do things halfway through. Like, activate `qiskit_sphinx_theme` rather than `sphinxcontrib.jquery`. If that better way truly matters, I'll go back and update the original PRs. Otherwise, I leave it for a follow-up PR.

> not that it matters too much I guess.

Yeah, it shouldn't matter. The goal is to get rid of all of this duplication. Tracked by https://github.com/Qiskit/qiskit_sphinx_theme/issues/323.